Sources/RSSwiftNetworking/Models/MultipartMedia.swift

```diff
@@ -19,6 +19,8 @@ public enum MimeType: String {
   case avi = "video/avi"
   case json = "application/json"
   
+  case usd, usdz = "application/octet-stream"
```

Comment:
I would create a single case representing the broader mime type to make the API clearer, this will leave other binary types out. i.e:
`case binary`
or 
`case octetStream`